packages/codemirror/src/editor.ts

```diff
@@ -99,6 +99,19 @@ const HOVER_TIMEOUT = 1000;
 // binding as it just introduces a lot of additional complexity without gaining anything.
 const USE_YCODEMIRROR_BINDING = true;
 
+export const usercolors = [
+  '#30bced',
+  '#6eeb83',
+  '#ffbc42',
+  '#ecd444',
+  '#ee6352',
+  '#9ac2c9',
+  '#8acb88',
+  '#1be7ff'
+];
+
+export const usernames = ['Io', 'Europa', 'Ganymede', 'Callistco'];
```

Comment:
`Callisto` (without extra `c`)? The good news there are many others that we can add, see this list: https://en.wikipedia.org/wiki/Moons_of_Jupiter#List

Reply:
@krassowski thanks for taking a look1 However, I will close this PR in favor of a better solution